test/servico/cst01Test.java

```diff
@@ -52,23 +51,25 @@ public void tearDown() {
     }
     
     @Test
-    public void testeNotificaçãoDeAusência_ÚnicoHorário() throws ParseException
-    {
+    public void testeNotificaçãoDeAusência_ÚnicoHorário() throws ParseException{

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/calebepb/substituicao/file/2738760059/issues/source?bid=3308597&fileBranchId=3353768#l54)